harness/helpers.ts

```diff
@@ -31,19 +31,25 @@ export const getConnectionStringForProxy = (proxy: string) => {
 
 export const forEachSDK = (tests) => {
     // get the list of SDK's and their capabilities
+    const SDKS_TO_TEST = process.env.SDKS_TO_TEST || global.JEST_PROJECT_SDK_TO_TEST
     let SDKs
     try {
-        SDKs = JSON.parse(process.env.SDKS_TO_TEST ?? '').map((sdk) => Sdks[sdk])
+        SDKs = JSON.parse(SDKS_TO_TEST ?? '').map((sdk) => Sdks[sdk])
     } catch (e) {
-        if (process.env.SDKS_TO_TEST && Sdks[process.env.SDKS_TO_TEST]) {
-            SDKs = [Sdks[process.env.SDKS_TO_TEST]]
+        if (SDKS_TO_TEST && Sdks[SDKS_TO_TEST]) {
+            SDKs = [Sdks[SDKS_TO_TEST]]
         } else {
             console.log('No specified SDKs to test, running all tests')
             SDKs = Object.values(Sdks)
         }
     }
     const scope = getServerScope()
 
+    if (process.env.SDKS_TO_TEST && global.JEST_PROJECT_SDK_TO_TEST && process.env.SDKS_TO_TEST !== global.JEST_PROJECT_SDK_TO_TEST) {
```

Comment:
isn't SDKS_TO_TEST an array? so wont this case always get hit

Reply:
the environment variable is a string

Reply:
oh but yes it would be unequal if its a stringified array

Reply:
Oh it can be an array or a string, I forgot

Reply:
as in, a single SDK not in a stringified array

Reply:
yep